src/plugins/telemetry/server/routes/telemetry_usage_stats.ts

```diff
@@ -23,59 +23,65 @@ export function registerTelemetryUsageStatsRoutes(
-      // security API directly to check privileges for this action. Note that the 'decryptedTelemetry' API privilege string is only
-      // granted to users that have "Global All" or "Global Read" privileges in Kibana.
-      const { checkPrivilegesWithRequest, actions } = security.authz;
-      const privileges = { kibana: actions.api.get('decryptedTelemetry') };
-      const { hasAllRequested } = await checkPrivilegesWithRequest(req).globally(privileges);
-      if (!hasAllRequested) {
-        return res.forbidden();
+      const security = getSecurity();
+      // We need to check useRbacForRequest to figure out if ES has security enabled before making the privileges check
+      if (security && unencrypted && security.authz.mode.useRbacForRequest(req)) {
+        // Normally we would use `options: { tags: ['access:decryptedTelemetry'] }` in the route definition to check authorization for an
+        // API action, however, we want to check this conditionally based on the `unencrypted` parameter. In this case we need to use the
+        // security API directly to check privileges for this action. Note that the 'decryptedTelemetry' API privilege string is only
+        // granted to users that have "Global All" or "Global Read" privileges in Kibana.
+        const { checkPrivilegesWithRequest, actions } = security.authz;
+        const privileges = { kibana: actions.api.get('decryptedTelemetry') };
+        const { hasAllRequested } = await checkPrivilegesWithRequest(req).globally(privileges);
+        if (!hasAllRequested) {
+          return res.forbidden();
+        }
       }
-    }
 
-    try {
-      const statsConfig: StatsGetterConfig = {
-        unencrypted,
-        refreshCache: unencrypted || refreshCache,
-      };
+      try {
+        const statsConfig: StatsGetterConfig = {
+          unencrypted,
+          refreshCache: unencrypted || refreshCache,
+        };
 
-      const body: v2.UnencryptedTelemetryPayload = await telemetryCollectionManager.getStats(
-        statsConfig
-      );
-      return res.ok({ body });
-    } catch (err) {
-      if (isDev) {
-        // don't ignore errors when running in dev mode
-        throw err;
+        const body: v2.UnencryptedTelemetryPayload = await telemetryCollectionManager.getStats(
+          statsConfig
+        );
+        return res.ok({ body });
+      } catch (err) {
+        if (isDev) {
+          // don't ignore errors when running in dev mode
+          throw err;
+        }
+        if (unencrypted && err.status === 403) {
+          return res.forbidden();
+        }
+        // ignore errors and return empty set
+        return res.ok({ body: [] });
       }
-      if (unencrypted && err.status === 403) {
-        return res.forbidden();
-      }
-      // ignore errors and return empty set
-      return res.ok({ body: [] });
+    } catch (err) {
+      logger.error(err);
+      throw err;
```

Comment:
DO NOT MERGE! We shouldn't log the error like this. It's only to confirm the error.